src/index.ts

```diff
@@ -91,41 +91,40 @@ export class XrpSettlementEngine {
     this.app.use(this.router.routes())
   }
 
-  public async start () {
-    console.log('STARTING TO LISTEN ON', this.port)
+  public async start() {
     this.server = this.app.listen(this.port)
     await this.rippleClient.connect()
     await this.subscribeToTransactions()
   }
 
-  public async shutdown () {
+  public async shutdown() {
     await Promise.all([
       this.server.close(),
       this.rippleClient.disconnect()
     ])
   }
 
-  private setupRoutes () {
-    this.router.post('/accounts', (ctx) => createAccount(ctx))
+  private setupRoutes() {
+    this.router.post('/accounts/:id', (ctx) => createAccount(ctx))
```

Comment:
This is an unusual pattern to use.
A POST to a REST endpoint shouldn't use path params.

The usual pattern is for the server to generate an ID and for this to be returned in the response so the client knows where to GET the resource afterwards.

In our case the server uses a property of the provided resource as the ID.

I know this is nit-picky but we might as well stick to conventions unless we have good reasons not to.

Reply:
I was going to raise this point as well. The connector is creating an account on the SE with a given id. To follow rest convention it should be a POST request with a json body of 
```json
{
 "id": "accountId"
}
```

Reply:
It could also be `PUT /accounts/:id` instead, although it is a bit unusual to send a PUT request with an empty body